.schema/pipeline/authorizers.remote_opa.schema.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Revert, see https://github.com/ory/oathkeeper/blob/master/.schemas/README.md :) Changes should be made here: https://github.com/ory/oathkeeper/tree/master/.schema/